src/monodroid/jni/monodroid-glue.cc

```diff
@@ -99,11 +98,17 @@ using namespace xamarin::android::internal;
 // implementation details as it would prevent mkbundle from working
 #include "mkbundle-api.h"
 
+#if !defined (NET6)
 #include "config.include"
+#endif
 #include "machine.config.include"
```

Comment:
Do we still need `machine.config.xml`?

Reply:
it contains a bunch of settings regarding pub tokens, remoting configuration etc - not **sure** we need it, but it might be needed for some things around the runtime